docs/programs-to-try.md

```diff
@@ -58,12 +58,12 @@ easily create and publish new Python packages
 pipx run create-python-package
 ```
 
-### flake8
```

Comment:
We should keep flake8 here (and add ruff separately), this is just “programs to try” and having flake8 here is correct.

Reply:
FIxed.